encoding/protobuf/testdata/istio.io/api/networking/v1alpha3/gateway.proto

```diff
@@ -218,7 +218,7 @@ message Gateway {
   // REQUIRED: One or more labels that indicate a specific set of pods/VMs
   // on which this gateway configuration should be applied. The scope of
   // label search is restricted to the configuration namespace in which the
-  // the resource is present. In other words, the Gateway resource must
+  // resource is present. In other words, the Gateway resource must
```

Comment:
Perhaps leave these testdata files untouched - we don't write nor maintain this code, nor is the documentation part of the CUE project, as they are just test inputs.